adapters/algorix/algorix.go

```diff
@@ -0,0 +1,170 @@
+package algorix
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type AlgoriXAdapter struct {
+	EndpointTemplate template.Template
+}
+
+// Builder builds a new instance of the Foo adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	template, err := template.New("endpointTemplate").Parse(config.Endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse endpoint url template: %v", err)
+	}
+	bidder := &AlgoriXAdapter{
+		EndpointTemplate: *template,
+	}
+	return bidder, nil
+}
+
+// MakeRequests Make Requests
```

Comment:
IMHO This comment doesn't provide any useful context. If you'd like to have a comment, please make it more descriptive. The `MakeRequests` method is common with the app so if you don't want to add comments, there is no pressure to do so.